spec/app1_spec.rb

```diff
@@ -60,4 +64,48 @@
       expect(chef_run).to create_systemd_service(s)
     end
   end
+
+  ###########################################################################
+  # FENESTRA
+  ###########################################################################
+
+  before do
+    stub_data_bag_item('osl-app', 'fenestra').and_return(
+      secret_key_base: '7eef5c70ecb083192f46e601144f9d77c9b66061b634963a507'\
+        '0fb086ae78bc9353af2c6311edb168abbb9d0bd428f800a0b1713534cf4ad239e8d'\
+        '07fdd16c34'
+    )
+  end
+
+  it 'should create systemctl privs for fenestra-staging' do
+    expect(chef_run).to install_sudo('fenestra-staging').with(
+      commands: ['/usr/bin/systemctl enable fenestra-staging_service',
+                 '/usr/bin/systemctl disable fenestra-staging_service',
+                 '/usr/bin/systemctl stop fenestra-staging_service',
+                 '/usr/bin/systemctl start fenestra-staging_service',
+                 '/usr/bin/systemctl status fenestra-staging_service',
+                 '/usr/bin/systemctl reload fenestra-staging_service',
+                 '/usr/bin/systemctl restart fenestra-staging_service'],
+      nopasswd: true
+    )
+  end
+
+  it 'should create systemctl privs for fenestra-production' do
+    expect(chef_run).to install_sudo('fenestra-production').with(
+      commands: ['/usr/bin/systemctl enable fenestra-production_service',
+                 '/usr/bin/systemctl disable fenestra-production_service',
+                 '/usr/bin/systemctl stop fenestra-production_service',
+                 '/usr/bin/systemctl start fenestra-production_service',
+                 '/usr/bin/systemctl status fenestra-production_service',
+                 '/usr/bin/systemctl reload fenestra-production_service',
+                 '/usr/bin/systemctl restart fenestra-production_service'],
+      nopasswd: true
+    )
+  end
+
+  %w(fenestra-staging-unicorn fenestra-production-unicorn).each do |s|
```

Comment:
In the recipe you call these `fenestra-staging_service` and `fenestra-production_service`